tox.ini

```
@@ -18,6 +18,7 @@ commands =
 sitepackages = False
 commands =
   flake8 {posargs}
+  black --check {posargs} .
```

I would change this to be:

```suggestion
  black --check {posargs} stestr tools docs setup.py
```

mostly because with `.` it might pick up on venvs from tox or other random stuff in the repo which could take some time if it contains external packages that don't use black. I think having it cover explicitly the paths in the repo which contain python code makes the most sense.

---

done